specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/machineLearningServices.json

```diff
@@ -6003,6 +6204,395 @@
+          "type": "string"
+        },
+        "prelabelAccuracyThreshold": {
+          "format": "double",
+          "description": "Prelabel accuracy threshold used in MLAssist feature.",
+          "type": "number"
+        },
+        "mlAssistEnabled": {
+          "description": "Indicates whether MLAssist feature is enabled.",
+          "type": "boolean"
+        }
+      }
+    },
+    "LabelingJobMediaProperties": {
+      "description": "Media specific properties in a labeling job.",
+      "type": "object",
+      "properties": {
+        "mediaType": {
+          "description": "Media type of data asset.",
+          "type": "string",
+          "enum": [
+            "Image",
+            "Text"
+          ],
+          "x-ms-enum": {
+            "name": "MediaType",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "ProgressMetrics": {
+      "description": "Progress metrics for a labeling job.",
+      "type": "object",
+      "properties": {
+        "totalDatapointCount": {
+          "format": "int64",
+          "description": "The total datapoint count.",
+          "type": "integer",
+          "readOnly": true
+        },
+        "completedDatapointCount": {
+          "format": "int64",
+          "description": "The completed datapoint count.",
+          "type": "integer",
+          "readOnly": true
+        },
+        "skippedDatapointCount": {
+          "format": "int64",
+          "description": "The skipped datapoint count.",
+          "type": "integer",
+          "readOnly": true
+        },
+        "incrementalDatasetLastRefreshTime": {
+          "format": "date-time",
+          "description": "The time of last successful incremental dataset refresh in UTC.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "StatusMessage": {
+      "description": "Active message associated with project.",
+      "type": "object",
+      "properties": {
+        "level": {
+          "description": "Severity level of the status message.",
+          "type": "string",
+          "enum": [
+            "Error",
+            "Information",
+            "Warning"
+          ],
+          "x-ms-enum": {
+            "name": "StatusMessageLevel",
+            "modelAsString": true
+          }
+        },
+        "code": {
+          "description": "Service-defined message code.",
+          "type": "string",
+          "readOnly": true
+        },
+        "message": {
+          "description": "A human-readable representation of the message code.",
+          "type": "string",
+          "readOnly": true
+        },
+        "createdTimeUtc": {
+          "format": "date-time",
+          "description": "Time in UTC at which the message was created.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "LabelingJob": {
```

Comment:
Would you mind if we could change this model name to `LabelingJobProperties` to reduce the confusion? Since this is only the properties rather than the resource itself

Reply:
Renamed to `LabelingJobProperties`.